assets/_scss/elements/_list.scss

```diff
@@ -21,7 +21,7 @@ article {
     }
```

Comment:
Yea I agree with @mjovel, I think styled lists should be the default, and use a class for the unstyled lists. I think I remember it being what I've seen in other libraries.